doc/classes/Particles2D.xml

```diff
@@ -282,43 +276,62 @@
 			<description>
 			</description>
 		</method>
-	</methods>
-	<members>
+	      </methods>
+	      
+	<members>			  
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
+		  The amount of particles that will be spawned on each emission
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
+		  The particle list's drawing order 
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+		  If true the emitter is currently emitting, if false it is not
```

Comment:
Should be "If [code]true[/code]" per style guide.  Remove everything after the comma, it is redundant.